portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,34 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world Michaela!</h1>");
+    ArrayList<String> comments = new ArrayList<>();
```

Comment:
Couple of comments here - one actionable and one for future thought:

1. Per [effective java item 64](go/ej3e-64) (if that go link doesn't resolve I believe i shared the PDF with the pod previously): refer to objects by their interfaces:
```suggestion
    List<String> comments = new ArrayList<>();
```
2. Have a gander at [Guava's ImmutableCollections](https://github.com/google/guava/wiki/ImmutableCollectionsExplained) - no need to change anything now, but these types (and guava, generally) will probably be useful for the capstone project.